src/runtime-support.cc

```diff
@@ -444,6 +445,22 @@ static void _data_in(const char* data, uint64_t len, std::optional<uint64_t> off
     }
 }
 
+void rt::close_connection(){
+    auto cookie = static_cast<Cookie*>(hilti::rt::context::cookie());
+    assert(cookie);
+
+    if ( auto c = std::get_if<cookie::ProtocolAnalyzer>(cookie) ) {
+        if (::zeek::session_mgr){
```

Comment:
The session manager is always available in Zeek, just so an `assert(::zeek::session_mgr)` should do it here.